plugins/binaryninja/binjastub/requirements.txt

```diff
@@ -1,6 +1,2 @@
-git+https://github.com/trailofbits/manticore.git@2918710d468c5c2225ac64c0b550a8c1c672b332
+manticore[native] @ git+https://github.com/trailofbits/manticore.git@04cc68dbf164af976df23e3a435207aa8fcb246d
```

Comment:
Can you add a comment (starting with `#`) to specify where this commit came from? Like maybe
```suggestion
# Latest commit on Manticore `master` branch as of July 28, 2022
manticore[native] @ git+https://github.com/trailofbits/manticore.git@04cc68dbf164af976df23e3a435207aa8fcb246d
```